.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Contact Tenzir
```

```suggestion
  - name: Contact Us
```

Maybe that feels a more personal?!